data/856/832/01/85683201.geojson

```diff
@@ -467,14 +467,20 @@
         "digitalenvoy:metro_code":250084,
         "eurostat:nuts_2021_id":"FRL06",
         "fips:code":"FRB8",
+        "fra-insee:code":"9384",
         "gn:id":2970554,
         "gp:id":12597203,
         "hasc:id":"FR.PR.VC",
+        "iso:code":"FR-84",
         "unlc:id":"FR-84",
         "wd:id":"Q12792"
     },
+    "wof:concordances_official":"fra-insee:code",
```

Comment:
Same comment from the US repo applies here.. seems like sometime we're saying concordance X is "official" in some records, but official alt in others.

Reply:
For a given country I've tried to always prefer the national statistical agency's IDs... but in some odd cases individual records won't have an ID. 

For some countries I was easily able to research what the ID should be (like new US `county` features), but in other cases that was prohibitive, so I backfilled the official with another known ID in our set of 4 preferred and variant sources so it would at least have an ID, and the key of that ID could still be noted as being different than it's peers. 

More detail in:

https://github.com/whosonfirst-data/whosonfirst-data/issues/2166#issuecomment-1741085713

Reply:
Here for the French departments, ISO does track `subdivision` codes, and since ISO is one of our preferred or variant "official" concordance sources I put that into `alt`, too.